addons/html_builder/static/src/plugins/language_selector_option.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+<t t-name="html_builder.LanguageSelectorOption">
+    <BuilderRow label.translate="Style" expand="true">
+        <BuilderSelect preview="false">
+                <BuilderSelectItem
+                    action="'websiteConfig'"
+                    actionParam="{
+                        views: ['website.header_language_selector']
+                    }">
+                    Dropdown
+                </BuilderSelectItem>
+                <BuilderSelectItem
+                    action="'websiteConfig'"
+                    actionParam="{
+                        views: ['website.header_language_selector', 'website.header_language_selector_inline']
+                    }">
+                    Inline
+                </BuilderSelectItem>
+        </BuilderSelect>
+        <t t-set-slot="collapse">
+            <BuilderRow label.translate="Label" level="1">
+                <BuilderSelect preview="false">
```

Comment:
you can move                         action="'websiteConfig'" on the BuilderSelect